python/tk_multi_workfiles/file_save_form.py

```diff
@@ -301,9 +319,10 @@ def _generate_path(self, env, name, version, use_next_version, ext, require_path
 
         # first make  sure the environment is complete:
         if not env or not env.context:
-            raise TankError("Please select a work area to save into...")
-        elif not env.work_template:
-            raise TankError("The working directory has not been defined for this work area.  Please select another work area.")
+            raise TankError("Please select a work area to save into.")
+        if not env.are_templates_configured():
```

Comment:
Possible to have a more specific error that mentions which templates explicitly?
Or change are_tempaltes_configured into something like ```assert_templates_are_configured```?